src/components/slider/index.tsx

```diff
@@ -583,6 +592,7 @@ export default class Slider extends PureComponent<SliderProps, State> {
                 styles.track,
                 trackStyle,
                 styles.minimumTrack,
+                Constants.isRTL && this.props.disableRTL && styles.disableRTL,
```

Comment:
I suggest renaming `styles.disableRTL` to something more specific like `styles.trackDisabledRTL` just to separate it from the actual prop and also because it's probably relevant only to the track element style

Reply:
Done :)